pysoa/server/server.py

```diff
@@ -372,9 +372,16 @@ def main(cls):
             except ImportError as e:
                 raise ValueError('Cannot import settings module %s: %s' % (cmd_options.settings, e))
             try:
-                settings_dict = getattr(settings_module, 'settings')
+                settings_dict = getattr(settings_module, 'SOA_SERVER_SETTINGS')
             except AttributeError:
-                raise ValueError('Cannot find settings variable in settings module %s' % cmd_options.settings)
+                try:
+                    settings_dict = getattr(settings_module, 'settings')
+                except AttributeError:
+                    raise ValueError(
+                        "Cannot find 'SOA_SERVER_SETTINGS' or 'settings' variable in settings module {}.".format(
+                            cmd_options.settings,
+                        )
+                    )
             settings = cls.settings_class(settings_dict)
```

Comment:
What should take precedence when both `SOA_SERVER_SETTINGS` and `settings` are present?

Reply:
`SOA_SERVER_SETTINGS`, because it's more explicit ... `settings` could mean anything, but `SOA_SERVER_SETTINGS` is clearly SOA server settings.